concrete/views/panels/page/design.php

```diff
@@ -2,134 +2,117 @@
+            </div>
 
-            <div class="list-group-item list-group-item-header"><?=t('Theme')?></div>
             <?php
-            foreach ($themes as $th) {
-                $selected = false;
-                if (is_object($selectedTheme) && $th->getThemeID() == $selectedTheme->getThemeID()) {
-                    $selected = true;
-                }
-                ?>
-                <div data-theme-id="<?=$th->getThemeID()?>" class="list-group-item ccm-page-design-theme-thumbnail <?php if ($selected) {
-    ?>ccm-page-design-theme-thumbnail-selected<?php
-}
-                ?> ">
-                    <span><i><?=$th->getThemeThumbnail()?>
-                        <?php if ($th->isThemeCustomizable()) {
-    ?>
-                        <span class="ccm-page-design-theme-customize">
-                            <a href="#" data-launch-panel-detail="page-design-customize" data-panel-detail-url="<?=URL::to('/ccm/system/panels/details/page/preview')?>" data-launch-sub-panel-url="<?=URL::to('/ccm/system/panels/page/design/customize', $th->getThemeID())?>">
-                                <?=t('Customize')?>
-                                <i class="fa fa-share"></i>
-                            </a>
-                        </span>
-                        <?php
-}
+            if (Config::get('concrete.marketplace.enabled')) {
                 ?>
-                    </i>
-                    <div class="ccm-panel-page-design-theme-description"><h4><?=$th->getThemeName()?></h4></div>
-
-                    </span>
+                <div class="ccm-marketplace-btn-wrapper">
+                <button type="button" onclick="window.location.href='<?= URL::to('/dashboard/extend/themes') ?>'" class="btn-info btn-block btn btn-large"><?= t("Get More Themes") ?></button>
                 </div>
-                <?php if ($selected) {
-    ?>
-                    <div class="list-group-item-collapse-wrapper">
                 <?php
-}
-                ?>
-            <?php
             }
-    ?>
-
-            <?php if ($selectedTheme) {
-    ?>
-                </div>
+            ?>
             <?php
-}
-    ?>
-            <a class="list-group-item list-group-item-collapse" href="#"><span><?=t('Expand')?></span></a>
-        </div>
-
-        <?php if (Config::get('concrete.marketplace.enabled')) {
-    ?>
-            <div class="ccm-marketplace-btn-wrapper">
-            <button type="button" onclick="window.location.href='<?=URL::to('/dashboard/extend/themes')?>'" class="btn-info btn-block btn btn-large"><?=t("Get More Themes")?></button>
-            </div>
-        <?php
-}
-    ?>
-
-    <?php
-} ?>
-
-    </div>
-</form>
+        }
+        ?>
 
+        </div>
+    </form>
 </section>
 
-    <div class="ccm-panel-detail-form-actions">
-        <button class="pull-right btn btn-success" type="button" data-panel-detail-action="submit"><?=t('Save Changes')?></button>
-    </div>
+<div class="ccm-panel-detail-form-actions side-panel">
```

Comment:
I don't think we need `side-panel` here. I'm not sure what that's doing other than adding additional padding but do we need that here? Why is this class called `side-panel` ?